docs/modules/release-notes/pages/5-3-3.adoc

```diff
@@ -0,0 +1,39 @@
+= 5.3.3 Release Notes
+
+== Enhancements
+
+* Improved the permission checks in the file connectors by adding a method that returns the permissions required to resolve field names.
```

Comment:
this is very diplomatic way of describing that a vulnerability was fixed, should we link CVE here?

Reply:
@k-jamroz , I couldn't find the CVE link in the relevant PRs or the Jira ticket. Is there a CVE ID?

Reply:
probably not yet: HZ-3405